.circleci/config.yml

```diff
@@ -25,7 +25,6 @@ jobs:
         name: Install bundle
         command: gem update --system && gem install bundler && bundle update
     - run: bundle exec rake circleci:build
-    - run: bundle exec rake test:coveralls
```

Comment:
Also delete the circleci:post run under test-ruby-2.5 above?

Reply:
Yeah, I thought I had done that. I will do that when I get back in front of a computer.

Reply:
Removed.